components/org.wso2.carbon.data.provider/src/main/java/org/wso2/carbon/data/provider/DataProviderAPI.java

```diff
@@ -1,162 +0,0 @@
-/*
```

Comment:
**Note**: Even though GitHub diff shows that this `DataProviderAPI` class is deleted here, in fact class was moved to the [`org.wso2.carbon.data.provider.internal`](https://github.com/wso2/carbon-analytics/pull/1226/files#diff-98ddc125858b02eb7fda1b7cc2770e90R19) package. See commit 8c78cfe83fec9180b34df5fa97a6b07c7e37fc95